src/components/SideMenu/MenuItem.tsx

```diff
@@ -90,7 +90,11 @@ export class OperationMenuItemContent extends React.Component<OperationMenuItemC
         deprecated={item.deprecated}
         ref={this.ref}
       >
-        <OperationBadge type={item.httpVerb}>{shortenHTTPVerb(item.httpVerb)}</OperationBadge>
+        {item.isWebhook ? (
+          <OperationBadge type="hook">hook</OperationBadge>
```

Comment:
I wonder why not `webhook`?

Reply:
Check out the screenshot above.`webhook` will be wider than the rest badges and will look ugly.

Reply:
This label/badge needs to be configurable. "hook" is a pejorative word in various vernacular.

Reply:
@adamaltman Do you have any other ideas? Maybe `whook` or `webh`? We use `opts` for `options` and `del` for `delete`.